src/forms/labels/labeled-field.js

```diff
@@ -72,10 +73,12 @@ const propTypes = {
   ...InputError.propTypes,
   children: PropTypes.node,
   hideErrorLabel: PropTypes.bool,
+  as: PropTypes.oneOf(['div', 'fieldset']),
 }
 
 const defaultProps = {
   hideErrorLabel: false,
+  as: 'div',
 }
 
 function LabeledField({
```

Comment:
Again, not new, but there are a few stories for this component with accessibility issues that can be seen in the storybook (form elements not having labels).

Reply:
👍 Issue created: https://github.com/LaunchPadLab/lp-components/issues/632